### ISSUE TITLE
Support local docker images when running cortex locally

### ISSUE THREAD

**Issue by deliahu** [cortexlabs/cortex#1094]
#### Description

Currently, predictor docker images must be hosted on a public registry (e.g. Dockerhub) or ECR. Attempting to use an image which exists only locally results in:

```text
error: ...: image: myrepo/cortex-python-predictor-cpu is not accessible
Error response from daemon: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

**Comment by RobertLucian** [comment]
@MrRace with #1114 you will be able to use local Docker images with your locally-deployed Cortex API(s). This will most likely land in `0.18`.

**Comment by MrRace** [comment]
> @MrRace with #1114 you will be able to use local Docker images with your locally-deployed Cortex API(s). This will most likely land in `0.18`.

Great! 
By the way, I find that when set multiple classifiers(e.t. by set multiple `-name`  in `cortex.yaml` ). Each time update one of them by `cortex deploy`, all the classifiers' service will update, but I just want to update the target classifier.  How can I do to just update the specific classifier? Maybe by some parameters？

**Comment by MrRace** [comment]
> @MrRace with #1114 you will be able to use local Docker images with your locally-deployed Cortex API(s). This will most likely land in `0.18`.

Great! 
By the way, I find that when set multiple classifiers(e.t. by set multiple `-name`  in `cortex.yaml` ). Each time update one of them by `cortex deploy`, all the classifiers' service will update, but I just want to update the target classifier.  How can I do to just update the specific classifier? Maybe by some parameters？

**Comment by deliahu** [comment]
@MrRace The reason all of the APIs are updated is because when Cortex deploys an API, it sends all files and directories (at and below the location of `cortex.yaml`) to the API container. If any of these files are changed, then it's possible that the API has changed, so it needs to redeploy the API.

One option is to create a separate directory for each API, each with a `cortex.yaml`, and deploy them separately. If your APIs share code and you would like to keep them in the same directory, you can also create a separate yaml file for each API (e.g. `api1.yaml`, `api2.yaml`, ...), and deploy them separately (e.g. `cortex deploy api1.yaml`, `cortex deploy api2.yaml`, ...).

**Comment by deliahu** [comment]
This has now been released in v0.18.0
